msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Use Microsoft Graph to fetch transcripts for a Teams meeting
-description: Describes the process, scenarios, and APIs to fetch transcripts in the post-meeting scenario.
+title: Use Microsoft Graph to fetch transcripts and recordings for a Teams meeting
+description: In this module, learn the process, scenarios, and APIs to fetch transcripts and recordings in the post-meeting scenario.
 ms.localizationpriority: high
 ms.topic: conceptual
 ---
-# Get meeting transcripts using Graph APIs
+# Get meeting transcripts and recordings using Graph APIs
 
-You can now configure your app to fetch Microsoft Teams meeting transcripts in the post-meeting scenario. Your app can use Microsoft Graph REST APIs to access and fetch transcripts generated for a Teams meeting that has been scheduled beforehand.
+You can now configure your app to fetch Microsoft Teams meeting transcripts and recordings in the post-meeting scenario. Your app can use Microsoft Graph REST APIs to access and fetch transcripts and recordings generated for a Teams meeting that has been scheduled beforehand.
 
-Here are some use cases for fetching meeting transcripts using Graph API:
+Here are some use cases for fetching meeting transcripts and recordings using Graph API:
 
 | Use case | How Transcript APIs help... |
```

Comment:
Needs to be changed to how Transcript and Recording APIs help..

Reply:
updated doc as suggested.